nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/nxql/TestNxqlConversion.java

```diff
@@ -1664,7 +1712,7 @@ protected void assertIn(String actual, String... expected) {
     }
 
     @Test
-    public void testConvertHintGeo() throws Exception {
+    public void testConvertHintGeo() {
```

Comment:
This cleanup makes the commit bigger than it needs to be, which slows down review. It should be in a separate commit.